build.gradle.kts

```diff
@@ -1,3 +1,7 @@
+
```

Comment:
Learning question - how is this build.gradle.kts different from the one that's in the `app` directory?

Reply:
Yeah so the different `build.gradle.kts` in each directory shows the module structure within the root project `build.gradle.kts`.

This in turn means that within a given Android gradle project, you will often have an `:app` submodule (with it's own `build.gradle.kts`), along with any other gradle submodules made, such as `:modules:readid:common`.